index.js

```diff
@@ -49,5 +93,34 @@ AFRAME.registerComponent('simple-link', {
    * Called when entity resumes.
    * Use to continue or add any dynamic or background behavior such as events.
    */
-  play() {}
+  play() {
+    this.updateEventListener();
+  },
+
+  updateEventListener() {
+    const { el } = this;
+    if (!el.isPlaying) {
+      return;
+    }
+    this.removeEventListener();
+    el.addEventListener(this.data.on, this.navigate);
+  },
+
+  removeEventListener() {
+    const {
+      data: { on }
+    } = this;
+    if (!on) {
+      return;
```

Comment:
Might be more straightforward to do this:

```javascript
if (on) {
 this.el.removeEventListener(on, this.navigate);
}
```

Reply:
Yup! Done.